app/views/candidate_mailer/declined_by_default.text.erb

```diff
@@ -1,3 +1,55 @@
+<% if FeatureFlag.active?('apply_again_email_content') && @last_course_choice_and_none_successful %>
+
+Dear <%= @application_form.first_name %>,
+
+# <%= 'Application'.pluralize(@declined_course_names.size) %> withdrawn automatically
+
+<% if FeatureFlag.active?('covid_19') %>
+You did not respond in time so we declined your <%= 'offer'.pluralize(@declined_course_names.size) %> for <%= @declined_course_names.to_sentence %>.
+<% else %>
+You did not respond within <%= @declined_courses.first.decline_by_default_days %> working days so we declined your <%= 'offer'.pluralize(@declined_course_names.size) %> for <%= @declined_course_names.to_sentence %>
+<% end %>
+
+<% if @rejected_course_count.zero? %>
```

Comment:
I wonder if we can make this logic clearer that this block is for anyone who hasn't received a rejection. Took me a few readings to parse.